Patches/Machines of War/Fuckyoweapon.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+  <Operation Class="PatchOperationFindMod">
```

Comment:
1. This entire file can be removed. The Marksman Rifle is already Patch Op Remove'd by an operation in the Stormcharger.
2. Why remove the Marksman Rifle?